globalSetup.ts

```diff
@@ -88,6 +88,16 @@ export default async function setup() {
           content: `
           [log]
           level = warn
+
+          [httpd]
+          socket_options = [{nodelay, true}]
+
+          [couchdb]
+          single_node = true
+
+          [cluster]
+          n = 1
+          q = 1
```

Comment:
I went through https://docs.couchdb.org/en/stable/maintenance/performance.html and enabled a few things. I don't think it had much of a performance impact, but it did get rid of some errors in the logs so I'm going to leave it in.